static/app/views/issueList/overview.tsx

```diff
@@ -615,14 +606,26 @@ class IssueListOverview extends Component<Props, State> {
           (group: BaseGroup) => group.issueCategory === IssueCategory.PERFORMANCE
         ).length;
 
-        const page = parseInt(this.props.location.query.page, 10);
+        const page = this.props.location.query.page;
+
+        const endpointParams = this.getEndpointParams();
+        const tabQueriesWithCounts = getTabsWithCounts(organization);
+        const currentTabQuery = tabQueriesWithCounts.includes(
+          endpointParams.query as Query
+        )
+          ? endpointParams.query
+          : null;
+        const tab = getTabs(organization).find(
+          ([tabQuery]) => currentTabQuery === tabQuery
+        )?.[1];
 
-        trackAdvancedAnalyticsEvent('issues_stream.count_perf_issues', {
+        trackAdvancedAnalyticsEvent('issues_tab.viewed', {
           organization,
-          page,
+          tab: tab ? tab.analyticsName : 'N/A',
```

Comment:
@0Calories I'd just make `tab` optional and supply an empty string or undefined.